docs/section5/Rule5-11.md

```diff
@@ -0,0 +1,31 @@
+
+# Envelope - Rule 5-11  
+
+**Rule ID:** 5-11  
+**Rule Description:**  Manual window shading devices such as blinds or shades are not required to be modeled.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_b in building_segment_b.thermal_blocks:```  
+
+  - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
+
+  - For each space in thermal zone: ```for space_b in zone_b.spaces:```  
+
+    - For each surface in space: ```for surface_b in space_b.surfaces:```  
+
+      - Get vertical glazings in exterior wall: ```if ( 60<= surface_b.tilt <= 90 ) AND ( surface_b.adjacent_to == "AMBIENT" ): windows_b = surface_b.fenestration_subsurfaces```  
+
```

Comment:
Few comments:
1. I think we need to remove "AND ( surface_b.adjacent_to == "AMBIENT"" and instead include all fenestraton on exterior or semi-exterior walls. E.g., if there is a window on a wall separating garage (an unenclosed space based on 90.1 defition) from adjacent condition space, the shading rules would apply to this window even though it's not adjacent to exterior.
2. As a side note, AMBIENT has been replaced with EXTERIOR.